src/data/actions/individualAssignment.js

```diff
@@ -0,0 +1,42 @@
+import {
+  INDIVIDUAL_ASSIGNMENT_REQUEST,
+  INDIVIDUAL_ASSIGNMENT_SUCCESS,
+  INDIVIDUAL_ASSIGNMENT_FAILURE,
+} from '../constants/individualAssignment';
+
+
+import EcommerceApiService from '../services/EcommerceApiService';
+
+const fetchIndividualAssignmentRequest = () => ({
+  type: INDIVIDUAL_ASSIGNMENT_REQUEST,
+});
+
+const fetchIndividualAssignmentSuccess = data => ({
+  type: INDIVIDUAL_ASSIGNMENT_SUCCESS,
+  payload: {
+    data,
+  },
+});
+
+const fetchIndividualAssignmentFailure = error => ({
+  type: INDIVIDUAL_ASSIGNMENT_FAILURE,
+  payload: {
+    error,
+  },
+});
+
+
+const fetchIndividualAssignment = options => (
+  (dispatch) => {
+    dispatch(fetchIndividualAssignmentRequest(options));
```

Comment:
It's not necessary to pass the `options` into this action as it doesn't use any of the form data (which is what the `options` variable will contain).